app/config/packages/twig.yml

```diff
@@ -7,7 +7,9 @@ twig:
         - '@ShopsysFramework/Admin/Form/fileuploadFields.html.twig'
         - '@ShopsysFramework/Admin/Form/imageuploadFields.html.twig'
         - '@ShopsysFramework/Admin/Form/pricetableFields.html.twig'
+        - '@ShopsysFramework/Admin/Form/orderItems.html.twig'
```

Comment:
again, link to the original PR would be probably helpful when someone was trying to understand this change